modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -145,20 +132,43 @@ The folder preview, like the file preview, is limited to the first ten lines of
 If a folder contains more than one file and the first file has more than ten lines, only the first ten lines of the first file will appear in the preview.
 
 [NOTE]
-====
 TigerGraph supports loading from archived and compressed GCS files directly.
 Currently supported file extensions include zip, tar.gz, tgz and tar.
 GraphStudio detects the file extension and automatically chooses the corresponding file format.
 If the file is encoded with one of these formats but has a non-standard file extension, you can manually specify the file format.
-====
 
 Confirm that the data has parsed correctly (see <<Configure the File Parser>>) and click btn:[ADD] to add the data source to your project, where it will appear in the working panel.
 
-Continue with <<Map data files to vertex type or edge type>>.
+--
+Azure Blob Storage::
++
+--
+TigerGraph creates a connection to ABS in order to access the file system in a blob.
+Only one file may be selected at a time.
+
+[NOTE]
+Data files, after decompression, must be in either CSV or Parquet format.
+If a file is in Parquet format, the data you are loading cannot use INT96 data types.
+
+TigerGraph supports loading from archived and compressed Blob files directly.
+Currently supported file extensions include zip, tar.gz, tgz and tar.
+GraphStudio detects the file extension and automatically chooses the corresponding file format.
+If the file is encoded with one of these formats but has a non-standard file extension, you can manually specify the file format.
+
+. After you click the btn:[ABS] data source icon, you will be prompted for your Connection String.
+. Add an alias to the data source connection.
+
```

Comment:
After adding the data source info, we don't do the authentication for their input connection string in GST. We should tell user this in documentation. They won't know if it works or not until trying to get sample data.